dev-lang/clojure/clojure-1.10.3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="General-purpose programming language with an emphasis on functional programming"
+HOMEPAGE="https://clojure.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${P}.tar.gz"
+
+LICENSE="EPL-1.0 Apache-2.0 BSD"
+SLOT="1.10.3"
```

Comment:
Why can't this be major slot? I believe 1.10 should stay.
Rather than hardoding it, you could use `ver_cut`.
https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-strings-for-manipulation